docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
 ```
 
-**Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
-:::note
- If running the SDK in a Server environment, session signatures may *not* be cached unless you provide an instance of `Storage` to the runtime.
- [Here](https://www.npmjs.com/package/node-localstorage) is an implementation of `LocalStorage` which creates local files to persist storage data.
- If storage is not available, session keys *MUST* be persisted in an external data store. 
- ```javascript
- const LocalStorage = require('node-localstorage').LocalStorage;
- const litNodeClient = new LitNodeClient({
-  litNetwork: "datil-dev",
-  debug: true,
-  storageProvider: {
-    provider: new LocalStorage('./storage.test.db'),
-  }
- });
- ```
-:::
-
-The `getSessionSigs()` function will try to create a session key for you and store it in local storage. You can also generate the session key yourself using `generateSessionKeyPair()` function and store it however you like. You can then pass the generated session key to `getSessionSigs()` as the `sessionKey` param.
-
-In the example above, we construct a SIWE message manually, but you can use the `checkAndSignAuthMessage` when in a browser context.
+**Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
 ## Resources You Can Request
 
-You can pass an array of "resource ability requests" to the `getSessionSigs()` function, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
+You can pass an array of `resourceAbilityRequests` to the above functions, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
 
-When session capability objects are omitted from the `getSessionSigs()` function call, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
+When session capability objects are omitted from functions generating session signatures, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
 
 ## Clearing Local Storage
```

Comment:
Should add an explanation of why you'd want to do this. Although I'm not sure why you'd want to outside of debugging

Reply:
Will say this in the overview